repo/packages/C/confluent-kafka/10/command.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
In `packagingVersion` 3.0, `command.json` is deprecated in favor of the `.cli` property of `resource.json`
This file can be removed.